browser/src/UI/Actions.ts

```diff
@@ -34,7 +34,7 @@ export interface IBufferUpdateAction {
     type: "BUFFER_UPDATE",
     payload: {
         id: number,
-        version: number,
+        modified: number,
```

Comment:
(It might be more intuitive if it is a `boolean` - it would mean we'd need to convert it from a `number` to a `boolean` somewhere in the pipe - either before the action creator or in the action creator). If `modified` is a `number`, and I'm just glancing at it, it makes me think there  might be multiple modified states or something - so making it a `boolean` would make this more crisp.